script/tempColorTokenBuild.ts

```diff
@@ -5,95 +5,105 @@ import {typeDefinitions, deprecatedJson, css, docJson, scss, javascript, typescr
 import type {ConfigGeneratorOptions, StyleDictionaryConfigGenerator} from '../types/StyleDictionaryConfigGenerator'
 import type {TokenBuildInput} from '../types/TokenBuildInput'
 
+const dirTokens = `src/tokens`
+const dirTokensBase = `${dirTokens}/base`
+const dirTokensFunctional = `${dirTokens}/functional`
+
 export const buildDesignTokens = (buildOptions: ConfigGeneratorOptions): void => {
   const themes: TokenBuildInput[] = [
     {
       filename: 'light',
       source: [
-        `tokens/functional/color/light/*.json5`,
-        `tokens/functional/shadow/light.json5`,
-        `tokens/functional/border/light.json5`
+        `${dirTokensFunctional}/color/light/*.json5`,
```

Comment:
I can totally see why you did this, but I think it's an optimisation at the expense of readability. I think I preferred the explicit paths as I didn't have to think at all about what it resolves to. Not blocking, but thought I'd share anyway.

Reply:
Hmm, I guess it may be harder to read. I am fine reverting this.